src/API/AbstractAPIClient.php

```diff
@@ -78,6 +78,17 @@ public function callAPI(Request $request)
         }
     }
 
+    /**
+     * @param Request $request
+     * @return Request
+     */
+    public function sanitizeHeaders(Request $request)
+    {
+        $request->removeHeader('Authorization');
+        $request->removeHeader('X-Auth-Key');
```

Comment:
I think auth email may also need to be here too.

It's PII and, given most installs will be on shared servers, it's probably best to include that with this patch.

Reply:
Actually, I think we can just remove the call entirely. We're double-logging.

Reply:
Removed `X-Auth-Email`.